create-react-native-app/src/index.js

```diff
@@ -61,11 +61,11 @@ function packageManagerType() {
   const defaultType = 'npm';
   const supportedTypes = ['yarn', 'npm'];
 
+  const supportedTypes = ['yarn', 'npm', 'pnpm'];
```

Comment:
Seems like you need to change line 62. You duplicated it and changed

Reply:
Whoops